README.md

```
@@ -1,40 +1,127 @@
+```yaml
+openai:
+api_key: your_openai_api_key
+
+google:
+client_id: your_google_client_id
+client_secret: your_google_client_secret
+```
+
+Google Cloud Consoleでは、承認済みのリダイレクトURIに以下を登録してください。
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)